### PR TITLE
Fix menu on mobile

### PR DESCRIPTION
**Pull request by FinalAngel**
Normal:
image

Tablet Portrait:
image

Mobile:
image

And showcase:
http://screencast.com/t/nOUBAzB2Lu
(Note: the white overlap has been already fixed)

### CODE REVIEW

**Comment by mkoistinen** [comment]
LGTM

File: cms/static/cms/sass/components/_toolbar.scss [hunk 1]
```diff
@@ -421,6 +421,53 @@
...
+
+    // overlap with logo when it gets too small
+    @media (max-width: $screen-tablet - 280px) {
+        margin-left: -110px;
+    }
+    // mobile only
+    @media (max-width: $screen-tablet) {
+        position: relative;
+        // remove spaces
+        font-size: 1px;
```
Comment (vxsx) [inline cms/static/cms/sass/components/_toolbar.scss]: why not `0`? :)

**Comment by vxsx** [comment]
LGTM

File: cms/static/cms/sass/components/_toolbar.scss (hunk 1 above)
Comment (FinalAngel) [inline cms/static/cms/sass/components/_toolbar.scss]: @vxsx AFAIK some browsers didn't support 0px back in the days?

**Pull request merged**